releases.md

```diff
@@ -0,0 +1,103 @@
+MAJOR.MINOR.PATCH `vX.Y`. The [support policy](#support-policy) applies to MAJOR and MINOR releases alike.
+
+## Release Tags and Branches
+
+Every time a Tekton project produces a release, a new tag `vX.Y.Z` is created, as well as a new branch that
+initially points to the git tag `vX.Y.Z`.
+
+- Supported releases: 
+    - branch name: `release-vX.N-lts`
+    - support window: until `release-vX.N+3`
+
+- Any other major or minor release:
+    - branch name: `release-vX.M`
+    - support window: until `release-vX.M+1`
+
+Once a release support window expires, the corresponding branch is deleted.
+It is recommended (but not mandatory) to make major releases coincide with supported ones.
+
+## Milestones
+
+Tekton projects may use [GitHub milestones][github-milestones] to plan their next release. This helps the
+maintainer and reviewer team to focus their work for the release and it gives users visibility on what they
+may expect for the upcoming releases.
+
+When used, milestone names must include the release number in `v<MAJOR>.<MINOR>` format.
+Once a release is made the milestone is closed. Patch releases are not tracked as part of the milestone.
+
+## Project Requirements
+
+Project must include release documentation in a `releases.md` file, which must include:
+
+- The release frequency adopted by the project
+- The first release where this [support policy](#support-policy) applies
+- A list of one year worth of releases. For each release:
+    - Number, name, date released, link to release notes
+    - End of life (EOL) date
+    - List of patch releases, each with link to release notes
+
+The document may include extra project-specific, user-facing release documentation.
+
+## Nightly Builds
+
+Tekton projects are encouraged to produce nightly builds, also called nightly releases.
+Nightly builds are produced for the benefit of Tekton users and developers but are not supported
+after they are produced. Users who rely on nightly builds must move to a newer nightly build or release to
+pick up any required bugfix, security patch or new feature. 
+
+Tekton projects always strive to keep the main branch in a consistent and usable state, however it may be
+possible for nightly builds to include partially implemented features and removal of deprecated features
+that will be thoroughly documented in the next upcoming release.
+
+The [plumbing][nightly-plumbing] repository contains the Tekton resources to be used to produce nightly
+builds. The plumbing repository includes examples of [nightly-build pipelines][nightly-pipeline] that projects
+may use to created their own. Nightly builds are triggered by [cronjobs][nightly-triggers] running
+in the *dogfooding* cluster.
+
+Each nightly build is associated to a git tag, which includes the date and short version of the git sha:
+
+```shell
+VERSION_TAG="v$(date +"%Y%m%d")-$(echo $GIT_SHA | cut -c 1-10)"
+```
+
+## Nightly releases
```

Comment:
This section confuses me a bit - it sounds like nightly releases are really just automated releases. That's great but I'm not sure we need to run those on a nightly basis.

Reply:
That's something we chatted about some time ago while discussing release frequency - I cannot find a link though, sorry. One option to be considered is to release every night instead of the current approach of monthly releases. That would require nightly builds to be a bit more formal i.e. release notes, probably a git tag.
What I wanted to say here is that even if a project was to adopt this model, it still would have to select three releases, give them a semantic version number and provide LTS for them.

If this is still too confusing I'm happy to remove this section for now.

Reply:
Ok that makes sense. (maybe adding this additional context would help)

Reply:
Added extra context